cassdegrees/templates/widgets/rulescripts.html

```diff
@@ -76,10 +76,15 @@
             <label>
                 Students must select...
             </label>
-            <input class="text" onkeydown="javascript: return checkKeys(event)" type="number" v-on:change="check_options" v-model="details.unit_count" min="0" step="6" max="1000" aria-required="true" required>
-            units
         </p>
 
+        <select v-model="details.list_type" v-on:change="check_options" required>
+            <option v-for="(msg, type) in list_types" v-bind:value="type">{{ msg }}</option>
+        </select>
+
+<!-- {#       instyle css only temporary #} -->
+        <input style="margin-left: 0;" class="text" onkeydown="javascript: return checkKeys(event)" type="number" v-on:change="check_options" v-model="details.unit_count" min="0" step="6" max="1000" required> units
```

Comment:
Sorry, didn't realise you had merged this yet - you can use !important to override css - e.g. `padding-left: 0 !important`.